docs/source/en/_toctree.yml

```diff
@@ -46,7 +46,8 @@
       title: Inference for multilingual models
     - local: generation_strategies
       title: Text generation strategies
-    - sections:
```

Comment:
Again this weird issue here that I saw also happened on #21400

Maybe @ydshieh knows why this is happening

Reply:
Hi, could you check on which of your branch commit that changes this line of this file? Got to say so far no idea from my side without looking the history.

Reply:
It seems to be [this commit](https://github.com/NielsRogge/transformers/commit/fca397c8341b35a6dcdb7440091a0c579eff0682)

Reply:
I fetch your branch, checkout to one commit before the one you shared above, and run `make fixup` or `python utils/check_doc_toc.py --fix_and_overwrite`, and the file `docs/source/en/_toctree.yml` is not changed.

Do you remember what command you ran? Otherwise, you can check the following list (in the next comment) of library versions to see if you have something different.

You can run `pip freeze` to get the list from your side.

Reply:
Here is mine

```bash
absl-py==1.3.0
accelerate==0.14.0
aiohttp==3.8.3
aiosignal==1.2.0
alembic==1.8.1
antlr4-python3-runtime==4.9.3
appdirs==1.4.4
APScheduler==3.9.1
arrow==1.2.3
asttokens==2.0.8
astunparse==1.6.3
async-timeout==4.0.2
attrs==22.1.0
audioread==3.0.0
autopage==0.5.1
Babel==2.10.3
backcall==0.2.0
backoff==1.11.1
backports.zoneinfo==0.2.1
beautifulsoup4==4.11.1
binaryornot==0.4.4
bitsandbytes==0.35.0
black==22.3.0
boto3==1.24.91
botocore==1.27.91
Brotli==1.0.9
cachetools==5.2.0
certifi @ file:///croot/certifi_1665076670883/work/certifi
cffi==1.15.1
chardet==5.0.0
charset-normalizer==2.1.1
chex==0.1.5
click==8.0.4
cliff==4.0.0
clldutils==3.12.0
cloudpickle==2.2.0
cmaes==0.8.2
cmd2==2.4.2
codecarbon==1.2.0
colorama==0.4.5
coloredlogs==15.0.1
colorlog==6.7.0
commonmark==0.9.1
contourpy==1.0.5
cookiecutter==1.7.3
cryptography==38.0.1
csvw==3.1.1
cycler==0.11.0
dash==2.6.2
dash-bootstrap-components==1.2.1
dash-core-components==2.0.0
dash-html-components==2.0.0
dash-table==5.0.0
datasets==2.9.0
decorator==5.1.1
decord==0.6.0
detectron2 @ git+https://github.com/facebookresearch/detectron2.git@7c2c8fb168a2093ce06a531c1208fba48d2984ec
# Editable install with no version control (diffusers==0.7.0.dev0)
-e /home/huggingface/diffusers
dill==0.3.4
distlib==0.3.6
dlinfo==1.2.1
dm-tree==0.1.7
docker==4.4.4
etils==0.8.0
evaluate==0.3.0
exceptiongroup==1.0.0rc9
execnet==1.9.0
executing==1.1.1
fairscale==0.4.12
faiss-cpu==1.7.2
fastjsonschema==2.16.2
filelock==3.8.0
fire==0.4.0
flake8==5.0.4
Flask==2.2.2
Flask-Compress==1.13
flatbuffers==2.0.7
flax==0.5.3
fonttools==4.37.4
frozenlist==1.3.1
fsspec==2022.8.2
fugashi==1.2.0
future==0.18.2
fvcore==0.1.5.post20220512
gast==0.4.0
gitdb==4.0.9
GitPython==3.1.18
google-auth==2.12.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
gql==3.4.0
graphql-core==3.2.3
greenlet==1.1.3.post0
grpcio==1.43.0
h5py==3.7.0
hf-doc-builder==0.4.0
huggingface-hub==0.11.1
humanfriendly==10.0
hydra-core==1.2.0
hypothesis==6.56.2
idna==3.4
importlib-metadata==4.13.0
importlib-resources==5.10.0
iniconfig==1.1.1
install==1.3.5
intel-extension-for-pytorch==1.11.0+cpu
iopath==0.1.9
ipadic==1.0.0
ipython==8.5.0
isodate==0.6.1
isort==5.10.1
itsdangerous==2.0.1
jax==0.3.6
jaxlib==0.3.5
jedi==0.18.1
Jinja2==3.1.2
jinja2-time==0.2.0
jmespath==1.0.1
joblib==1.2.0
jsonschema==4.16.0
jupyter-core==4.11.1
kenlm @ https://github.com/kpu/kenlm/archive/master.zip
keras==2.11.0
keras-nlp==0.4.0
Keras-Preprocessing==1.1.2
kiwisolver==1.4.4
kubernetes==12.0.1
language-tags==1.1.0
libclang==14.0.6
librosa==0.9.2
llvmlite==0.39.1
Mako==1.2.3
Markdown==3.4.1
MarkupSafe==2.1.1
matplotlib==3.6.1
matplotlib-inline==0.1.6
mccabe==0.7.0
modelcards==0.1.6
mpmath==1.2.1
msgpack==1.0.4
multidict==6.0.2
multiprocess==0.70.12.2
mypy-extensions==0.4.3
natten==0.14.4
nbformat==5.7.0
nltk==3.7
numba==0.56.3
numpy==1.23.4
oauthlib==3.2.1
omegaconf==2.2.3
onnx==1.12.0
onnxconverter-common==1.12.2
onnxruntime==1.12.1
onnxruntime-gpu==1.12.1
onnxruntime-tools==1.7.0
opt-einsum==3.3.0
optax==0.1.3
optuna==3.0.3
packaging==21.3
pandas==1.5.0
parameterized==0.8.1
parso==0.8.3
pathspec==0.10.1
pbr==5.10.0
pexpect==4.8.0
phonemizer==3.2.1
pickleshare==0.7.5
Pillow==9.2.0
Pint==0.16.1
pkgutil_resolve_name==1.3.10
plac==1.3.5
platformdirs==2.5.2
plotly==5.10.0
pluggy==1.0.0
pooch==1.6.0
portalocker==2.0.0
poyo==0.5.0
prettytable==3.4.1
prompt-toolkit==3.0.31
protobuf==3.19.6
psutil==5.9.2
ptyprocess==0.7.0
pure-eval==0.2.2
py==1.11.0
py-cpuinfo==8.0.0
py3nvml==0.2.7
pyarrow==9.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycocotools==2.0.5
pycodestyle==2.9.1
pycparser==2.21
pyctcdecode==0.4.0
pydot==1.4.2
pyflakes==2.5.0
Pygments==2.13.0
pygtrie==2.5.0
pyknp==0.6.1
pynvml==11.4.1
pyOpenSSL==22.1.0
pyparsing==3.0.9
pyperclip==1.8.2
pypng==0.20220715.0
pyrsistent==0.18.1
pytesseract==0.3.10
pytest==7.1.3
pytest-forked==1.4.0
pytest-timeout==2.1.0
pytest-xdist==2.5.0
python-dateutil==2.8.2
python-slugify==6.1.2
pytz==2022.4
pytz-deprecation-shim==0.1.0.post0
PyYAML==6.0
ray==2.0.0
rdflib==6.2.0
regex==2022.9.13
requests==2.28.1
requests-oauthlib==1.3.1
requests-toolbelt==0.10.0
resampy==0.4.2
responses==0.18.0
rfc3986==1.5.0
rhoknp==1.1.2
rich==11.2.0
rjieba==0.1.11
rouge-score==0.1.2
rsa==4.9
s3transfer==0.6.0
sacrebleu==1.5.1
sacremoses==0.0.53
safetensors==0.2.1
scikit-learn==1.1.2
scipy==1.8.1
segments==2.2.1
sentencepiece==0.1.97
sigopt==7.5.0
six==1.16.0
smmap==5.0.0
sortedcontainers==2.4.0
soundfile==0.11.0
soupsieve==2.3.2.post1
SQLAlchemy==1.4.41
stack-data==0.5.1
stevedore==4.0.1
SudachiDict-core==20220729
SudachiPy==0.6.6
sympy==1.11.1
tabulate==0.9.0
tenacity==8.1.0
tensorboard==2.11.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
tensorboardX==2.5.1
tensorflow==2.11.0
tensorflow-estimator==2.11.0
tensorflow-hub==0.12.0
tensorflow-io-gcs-filesystem==0.29.0
tensorflow-probability==0.19.0
tensorflow-text==2.11.0
tensorstore==0.1.27
termcolor==2.0.1
text-unidecode==1.3
tf2onnx==1.8.4
threadpoolctl==3.1.0
timeout-decorator==0.5.0
timm==0.6.11
tokenizers==0.13.1
tomli==2.0.1
toolz==0.12.0
torch==1.13.0+cu116
torchaudio==0.13.0+cu116
torchtext==0.13.1
torchvision==0.14.0+cu116
tqdm==4.64.1
traitlets==5.4.0
-e git+https://github.com/huggingface/transformers@3113fc5307687362291ea5e9fcde723a54bf8978#egg=transformers
typer==0.7.0
typing_extensions==4.4.0
tzdata==2022.5
tzlocal==4.2
unidic==1.1.0
unidic-lite==1.0.8
uritemplate==4.1.1
urllib3==1.26.12
virtualenv==20.16.5
wasabi==0.10.1
wcwidth==0.2.5
websocket-client==1.4.1
Werkzeug==2.2.2
wrapt==1.14.1
xmltodict==0.13.0
xxhash==3.0.0
yacs==0.1.8
yarl==1.8.1
zipp==3.9.0

```

Reply:
@ydshieh providing such a long list is not really helpful, sorry 😅 which libraries are you suspecting to be the cause of this?

Reply:
Could you check `yaml`? If you can find it by `pip show yaml`, you can do

```
import yaml
print(yaml.__version__)
```
Mine is `6.0`.

(long list is not that difficult to check - just copy past to a file and use a diff tool)

Reply:
I don't have the `yaml` package installed on my system:
```
(env) niels@hf-dgx-01:~/python_projects/transformers$ pip show yaml
WARNING: Package(s) not found: yaml
```